### PR TITLE
Make templates usable for func create

### PR DESCRIPTION
**Pull request by shinigambit**
Signed-off-by: Fabian Lopez <lfabian@vmware.com>

## Which issue(s) this PR fixes

Partially fixes: #30

## What this PR solves
Makes templates usable by `kn func create`

## Details for release notes

```release-note
Create via kn-func can now make use the java and python templates from this repo:
kn func create -r https://github.com/vmware-tanzu/function-buildpacks-for-knative.git -l java -t (cloudevents-maven|cloudevents-gradle|http-maven|http-gradle)
kn func create -r https://github.com/vmware-tanzu/function-buildpacks-for-knative.git -l python -t (cloudevents|http)
```

## Describe testing done for PR
Created a fork for this repo and by making the same changes in this PR to the `manifest.yaml` I was able to create functions by pointing to my fork with the `-r` flag

## Special notes for your reviewer
As mentioned in the linked issue, there still is the problem of building the generated functions with our buildpacks

### CODE REVIEW

**Comment by shinigambit** [comment]
wondering if we should update the builders in the template from `us.gcr.io/daisy-284300/kn-fn/builder:0.0.6` to `ghcr.io/vmware-tanzu/function-buildpacks-for-knative/functions-builder:0.0.9`

**Comment by bbtong** [comment]
Manually ran the tests on `bt/test-pr-48` -- if you work off a branch and not a fork, you can trigger the testing suite yourself via Actions!

All the tests passed for me, but I'm going to tag @andrew-su to see if he remembers more context on the format of `manifest.yaml` and if we can wipe out all those other entries.

**Comment by andrew-su** [comment]
> wondering if we should update the builders in the template from `us.gcr.io/daisy-284300/kn-fn/builder:0.0.6` to `ghcr.io/vmware-tanzu/function-buildpacks-for-knative/functions-builder:0.0.9`

I think we definitely should update it to use new builder. (new PR)

**Comment by shinigambit** [comment]
My bad. @bbtong should I move the changes to a branch in this repo first before we can merge it?

**Comment by bbtong** [comment]
> My bad. @bbtong should I move the changes to a branch in this repo first before we can merge it?

It's fine, manually ran the tests! You're good to merge.

**Comment by andrew-su** [comment]
> see if he remembers more context on the format of manifest.yaml and if we can wipe out all those other entries.

Not a clue.

**Comment by bbtong** [comment]
@shinigambit I'm wondering about the removal of `runtimes` -- was that deprecated or just not needed now?

If it's not needed, you're good to merge; but if you could add a comment here or in the `manifest.yaml` with a link to the full spec that you used that would be great. It's been a while and I am assuming the spec has changed, but I'm seeing 404s in Kn-plugin-func's repo to this info.

**Review by bbtong — APPROVED** [review]

**Comment by shinigambit** [comment]
@bbtong sorry I missed your comment, there's apparently some logic in the cli that reads the `template` path (in our case "template") and derives the languages from to the directories in the `templates` path (java/python in our case) then inside the language directories it checks for runtimes; That currently matches our project structure so it doesn't seem to require additional configuration to work

**Pull request merged**
